### PR TITLE
[Code] Add IsCloseToBanker method

### PR DESCRIPTION
**Pull request by Akkadius**
# Description

This is a very small PR that adds a helper method in assistance to another PR https://github.com/EQEmu/Server/pull/4453

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist

- [x] I have performed a self-review of my code. Ensuring variables, functions and methods are named in a human-readable way, comments are added only where naming of variables, functions and methods can't give enough context.
- [x] I own the changes of my code and take responsibility for the potential issues that occur

### CODE REVIEW

**Review by Kinglykrab — APPROVED** [review]

**Comment by Kinglykrab** [comment]
Are we going to change the code he submitted to use this? Or just add the method itself?

File: zone/mob.cpp [hunk 1]
```diff
@@ -8611,3 +8611,15 @@ void Mob::SetExtraHaste(int haste, bool need_to_save)
 		}
 	}
 }
+
+bool Mob::IsCloseToBanker()
+{
+	for (auto &e: entity_list.GetCloseMobList(this)) {
+		auto mob = e.second;
+		if (mob->IsNPC() && mob->GetClass() == Class::Banker) {
```
Comment (Kinglykrab) [inline zone/mob.cpp:8619]: Should we validate `mob` here as well?

Comment (Akkadius) [reply zone/mob.cpp:8619]: Shouldn't have to but it doesn't hurt

**Comment by Akkadius** [comment]
> Are we going to change the code he submitted to use this? Or just add the method itself?

I'm just adding the method in this instance. Will follow up with the PR separately.

**Comment by Kinglykrab** [comment]
> > Are we going to change the code he submitted to use this? Or just add the method itself?
> 
> I'm just adding the method in this instance. Will follow up with the PR separately.

10-4.

**Pull request merged**